src/workers/event.rs

```diff
@@ -2367,8 +2422,8 @@ mod test {
                 issuer: "my-issuer-5".to_string(),
                 actors: vec![ActorDescription {
                     id: "jabba".to_string(),
-                    image_ref: None,
-                    name: None,
+                    image_ref: Some("jabba.tattoinecr.io/jabba:latest".to_string()),
```

Comment:
I'm sorry sir, it is _ackshually_ tatooine, not tattoine, unless you're looking for a Star Wars themed tattoo shop